code/modules/plumbing/plumbers/pill_press.dm

```diff
@@ -1,31 +1,37 @@
 ///We take a constant input of reagents, and produce a pill once a set volume is reached
 /obj/machinery/plumbing/pill_press
-	name = "pill press"
-	desc = "A press that presses pills."
+	name = "chem press"
```

Comment:
shouldnt this be chemical press or something, chem just feels like what players would call it

Reply:
Fair